news/2025/2025-07-28-osucatch-world-cup-2025-concludes.md

```diff
@@ -0,0 +1,462 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/6567640" style="background-image: url('/wiki/shared/avatars/Realmaas.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-loved" href="https://osu.ppy.sh/users/6567640">Realmaas</a></p>
+
+After this year's win, motivation seems to be at an all-time high in the Argentinian community. What can we expect from your team in the future?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/8172283" style="background-image: url('/wiki/shared/avatars/-Story-.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/8172283">-Story-</a></p>
+
+I'm very happy with everyone who congratulated us, and I'm not just talking about Argentina when I say that we truly received a LOT of support. It's undoubtedly a great memory for us, and it means a lot, even though there's some support and also hate but we choose to focus on the effort it took to get here, day after day.
+
+None of this would have been achieved without the motivation we had as a team, as a family, as a friendship. We went through a lot but I would like to thank the friends i have as colleagues. My best friends are [Zeus](https://osu.ppy.sh/users/8329151), [Alex](https://osu.ppy.sh/users/17738253), [VelperK](https://osu.ppy.sh/users/348467), [Naarkz](https://osu.ppy.sh/users/19490380), [Gonza](https://osu.ppy.sh/users/17309439) and I want to say thanks for everything to them and for the supporters that were there every single match and day to pray for our win ❤️ 
+
+Now... Regarding the Catch World Cup, I think it will depend on several factors that allow us to have or make time to play them, but we no longer have the pressure of being champions because we already are. Having fun and accepting that the game is a game is the key to understand that you don't have to take things too seriously and that no matter how much people judge you, whether you win or lose, it doesn't matter! You have to enjoy having teammates, moments where you fall and get up, situations where you don't feel like you're what you demonstrated in practice, and much more.
+
+Take the World Cup as a place to explore a more competitive and healthy environment where you can have fun and play with your friends.
+Don't overthink too much and enjoy what this is, just a game that we all love.
+
+See you next year, I'm hoping to play with all my friends again and again because they are the ones that motivate me to keep playing. 
```

Comment:
```suggestion
See you next year, I'm hoping to play with all my friends again and again because they are the ones that motivate me to keep playing.
```